dist.ini

```diff
@@ -1,69 +1,12 @@
 name    = Cache-RedisDB
-author  = binary.com <BINARY@cpan.org>
+author  = DERIV <DERIV@cpan.org>
```

Comment:
Had a doubt since i dont fully understand how the publishing works. The author is Deriv but it is released under binary.com on metacpan. Should it not be released under Deriv ?

Reply:
You are right. let me ask Tom.
but it  cannot be release under DERIV because the namepsace is used by BINARY account.
So 2 options here:
1.  set author as DERIV 
2. set author as binary
3

Reply:
asked, we should keep binary name
Will change it back to binary